src/routes/transactions/transactions.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
+import { AddressInfoHelper } from '../common/address-info/address-info.helper';
+import { MultisigTransactionMapper } from './mappers/multisig-transaction.mapper';
+import { TransactionsController } from './transactions.controller';
+import { TransactionsService } from './transactions.service';
+
+@Module({
+  controllers: [TransactionsController],
+  providers: [
+    AddressInfoHelper,
```

Comment:
Here the `AddressInfoModule` should be imported

Reply:
Changed in [6d58c66](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/6d58c665e46105677aa5f746b9ac461b4e0c8ee3)